City-of-Pori/pori-kada
# Feature/pori 78 page node structure

Manual changes needed: in page display settings, drag "Lead paragraph" right before "Body"

To test:
- Make sure main image and info box are gone from page (this is what the update hook should take care of)
- Left side menu should be gone from pages added to footer
- Make sure a lead paragraph ("ingressi") appears on the content page when one is added on the form
- Make sure the main menu is not duplicated on the content area on pages that are added as menu items
